WordPress/Classes/ViewRelated/Tools/PromptViewController.swift

```diff
@@ -0,0 +1,123 @@
+import Foundation
+
+
+/// Protocol that defines the methods required by PromptViewController's Children(s).
+///
+@objc
+protocol PresentedViewController
```

Comment:
If anything, I'd say this is a `Presentable` view controller. Nothing is _presented_ until you present it ;)
However, I find this very ambiguous and doesn't really represent what this protocol is about.

Also, since the protocol can't impose a restriction saying that it only applies to `UIViewController` I'd drop the `ViewController` from the name.

If you look at the requirements, this is about an object that supports saving or canceling. My first guess at a name would be `Saveable` although it doesn't include canceling.

Also, this is showing a `Done` button not a `Save` button. As you mentioned on Friday, this pattern might require tweaking the labels at some point and having (Done/Save/Send/...). What all of those say to me is that there's a primary action that commits whatever is inside, and a canceling action that discards it.

How about `Confirmable`? "Confirmation" also implies that there's an option to reject.